microsoft-edge/webview2/get-started/winui2.md

```diff
@@ -8,18 +8,14 @@ ms.prod: microsoft-edge
 ms.technology: webview
 ms.date: 04/27/2022
 ---
-# Get started with WebView2 in WinUI 2 (UWP) apps (public preview)
+# Get started with WebView2 in WinUI 2 (UWP) apps
```

Comment:
When I looked at the preview URL, the sidebar content lists this page as a preview. Does that need to be updated?

![image](https://user-images.githubusercontent.com/8368026/177378002-6f688e43-2e50-46fd-8137-e430d91c219d.png)

Reply:
I plan to update the usual 3-5 spots for updating article title, @jasonstephen15 - I can take care of these kind of mechanics (vs content gathering)

Reply:
fixed globally